config/lambdatest/wdio.android-browser-emulators.app.conf.ts

```diff
@@ -0,0 +1,51 @@
+import { config as baseConfig } from '../wdio.shared.conf.js';
```

Comment:
Can you structure it the same as the SL configs? Most info here is double and can be retrieved from the shared config or be put in a separate LT shared config

Secondly, the name of the file is confusing, app and browser have a different config, and this can't be seen in the name